var/spack/repos/builtin/packages/gaudi/package.py

```diff
@@ -23,25 +29,40 @@ class Gaudi(CMakePackage):
     variant('vtune', default=False,
             description='Build with Intel VTune profiler support')
 
+    # only build subdirectory GaudiExamples when +tests
+    patch("build_testing.patch", when="@:33.1")
+    # fix for the new cmake config, should be merged in branch
+    patch('python2.patch', when="@develop")
+    # fixes for the cmake config which could not find newer boost versions
+    patch("link_target_fixes.patch", when="@33.0:33.1")
+    patch("link_target_fixes32.patch", when="@:32.2")
+
     # These dependencies are needed for a minimal Gaudi build
     depends_on('boost@1.67.0: +python')
     depends_on('cmake', type='build')
+    depends_on('cppgsl')
     depends_on('intel-tbb')
     depends_on('libuuid')
-    depends_on('python@:2.99.99')
+    # some bugs with python 3.8
+    depends_on('python@:3.7.99')
+    depends_on('py-setuptools@:45.99.99', when='^python@:2.7.99')
+    depends_on('py-six')
     depends_on('py-xenv@develop_2018-12-20:')
     depends_on('range-v3')
     depends_on('root +python +root7 +ssl +tbb +threads')
     depends_on('zlib')
 
+    depends_on('py-nose', when="@develop")
+
     # These dependencies are required by the Gaudi test suite
     depends_on('aida', when='+tests')
```

Comment:
```suggestion
    depends_on('aida', type='test')
```
Same for all other test deps